src/apis/twitchApi.js

```diff
@@ -7,40 +7,35 @@ const twitchUrl = 'https://api.twitch.tv/helix';
 async function getViewers() {
     try {
         const req = await axios.get(`http://tmi.twitch.tv/group/user/${config.channel}/chatters`);
-        const chatters = req.data.chatters;
-
-        return _.uniq([
-            ...chatters.broadcaster,
-            ...chatters.vips,
-            ...chatters.moderators,
-            ...chatters.staff,
-            ...chatters.admins,
-            ...chatters.global_mods,
-            ...chatters.viewers,
-        ]);
+        const chatters = _.flatMapDeep(req.data.chatters);
+        return _.uniq(chatters);
     } catch (error) {
         console.log('twitch.getViewers', error);
     }
 }
 
-async function getUserByUsername(username) {
-    const req = await axios.get(`${twitchUrl}/users?login=${username}`, {
-        headers: { 'Client-ID': process.env.TWITCH_CLIENT },
+async function getUsersByUsernames(usernames) {
```

Comment:
You can make this a spread operator if helps with no having to pass around arrays?

```js
function a (...b) { console.log(b) }

a(1);
a(1, 2, 3, 4, 5);
a(...[1, 2, 3, 4, 5])
```